src/hotspot/share/gc/g1/g1Analytics.hpp

```diff
@@ -139,10 +139,9 @@ class G1Analytics: public CHeapObj<mtGC> {
 
   double predict_concurrent_refine_rate_ms() const;
   double predict_dirtied_cards_rate_ms() const;
-  double predict_young_card_merge_to_scan_ratio() const;
-
-  double predict_mixed_card_merge_to_scan_ratio() const;
 
+  // Predict how many of the given remembered set of length rs_length will add to
+  // the number of total cards scanned.
```

Comment:
I'm having a very hard time parsing that comment.  I _think_ it is equivalent to "Predict how many cards in a remembered set of length rs_length will need to be scanned."

Reply:
This predictor is a bit complicated: it predicts for that given remembered set length, what part of it adds to the number of scanned cards - assuming that the pending log buffers are always scanned, we need to find out how many additional cards the remembered set will dirty (and need to be scanned). I.e. the remembered set cards that do not duplicate the log buffer cards.
I tried to reformulate that, please let me know if it is more understandable now.